### PR TITLE
Lafe 86 create link flap test on dut

### PR DESCRIPTION
**Pull request by joej164**
created a script to bounce the interface as fast as I reasonably can without writing a C program.

copied the 'run_command_with_loganalyzer.yml' file and converted it to a `run_script...' file so I could verify no errors in the syslog as requested.

copied the cpu and memory checks from my log_flood test which were from another test and used those to verify the system wasn't wonky.

### CODE REVIEW

File: ansible/roles/test/tasks/sonic_link_flap.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+  - name: post test memory values
+    debug:
+      var: used_memory_after.stdout
+
+  - name: post test cpu values
+    debug:
+      var: shell_cpu_usage_after.stdout
+
+  # Arbitrarially picked that if the CPU is 10% 10 seconds after the log flood is done, should fail
```
Comment (joeslazaro) [inline ansible/roles/test/tasks/sonic_link_flap.yml]: This references 'log flood'. Did you forget to edit this after copying it from elsewhere?

**Review by joeslazaro — CHANGES_REQUESTED** [review]
Just one possible issue with a comment to look at. Everything else looks okay.

File: ansible/roles/test/tasks/sonic_link_flap.yml (hunk 1 above)
Comment (joej164) [reply ansible/roles/test/tasks/sonic_link_flap.yml]: yup, will fix that

**Review by joeslazaro — APPROVED** [review]
The newest changes look fine.

**Pull request merged**